### ISSUE TITLE
Improve accessibility (part 1)

### ISSUE THREAD

**Issue by HugoGresse** [HugoGresse/open-feedback#1014]
- [x] landing page: no header/main/footer/navigation
- [x] feedback app: 
    - [x] no header/main/footer
    - [x] animated image does not appear on safari
    - [x] vote items should be buttons
    - [x] logo should have the event name in alt
    - [x] talks list <a> missing title
    - [x] why is there two spans for votes count? 
    - [x] Error message should be aria-polite ? + aria-invalid
- [ ] admin app: 
    - [x] header ok, no sidebar + nav, main, footer
    - [x] header should have a h2, not h5
    - [x] dashboard: card title should be h3
    - [x] dashboard: talks lsit in card should be list
    - [x] speakers and talks are list, not div
    - [x] moderation: list, not div (ul/li)
    - [x] talks/moderation: speakers on item are clicable, should be buttons
    - [x] event/theme: chip should be buttons
    - [x] event/theme: no alt on logo & favicon
    - [x] voting form: h3 not h6 on "Vote choices" 
    - [x] "Open event" no title
    - [x] Users: should be list
    - [x] SidePanel should autofocus on first field for talks/speakers/users
    - [x] Error message should be aria-polite ?  + aria-invalid
- [x] lang attr on <html> role=application on body
- [ ] enable cypress-axe 
- [x] check role everywhere 
- [ ] remercier Emmanuel
